cypress/features/regression/profileClassic.feature

```diff
@@ -20,7 +21,7 @@ Feature: Profile component
       And avatar is taken from "<avatar>"
     Examples:
       | email                | avatar                                                                        |
-      | andrew.tait@sage.com | https://www.gravatar.com/avatar/ec55ecf2e1c7e7e56a904b50245c24a4?s=40&d=blank |
+      | andrew.tait@sage.com | https://www.gravatar.com/avatar/ec55ecf2e1c7e7e56a904b50245c24a4?s=40&d=404 |
```

Comment:
Why did You change to 404 at the end of this line?

Reply:
because it works that way 👾